libs/backendless-console-api.js

```diff
@@ -305,6 +305,26 @@ class Backendless {
         return runInParallel(tasks, 10)
     }
 
+    getAppCustomApiKeys() {
+        console.log('Fetching custom API keys..')
+
+        return Promise.all(
+          filterLive(this.appList).map(app => {
+              app.id = app.id || app.appId
+
+              if (app.id) {
+                  return this.instance.get(`/${app.id}/console/appsettings`)
+                    .then(({data}) => {
+                        return app.apiKeys = data.apiKeys
+                          .filter(key => key.deviceType === 'CUSTOM')
+                          .map(key => key.name)
+                          .sort()
+                    })
+              }
+          })
+        )
+    }
```

Comment:
Can't we simply replicate an existing logic like this?
<img width="548" alt="image" src="https://github.com/Carrydmcc/bact/assets/784011/5dd62161-7b98-4306-975b-7d47129d30f9">